src/exec/mod.rs

```diff
@@ -195,43 +294,10 @@ impl CommandTemplate {
     fn generate(&self, input: &Path, path_separator: Option<&str>) -> Command {
         let mut cmd = Command::new(self.args[0].generate(&input, path_separator));
         for arg in &self.args[1..] {
-            cmd.arg(arg.generate(&input, path_separator));
+            cmd.try_arg(arg.generate(&input, path_separator)).unwrap();
```

Comment:
I haven't reviewed the full code, but it looks strange that we are forced to use `.unwrap()` here. Is it because we have used `args_would_fit` somewhere before? If so, is this really the best API for `argmax`? Or would it be enough if we had `try_args` that would leave the `Command` untouched in case it returns an error?

Reply:
Oh I don't think I meant to leave this unwrap() here, just hadn't finished converting everything to argmax

Reply:
Oh, I see. Sorry for the confusion.